jumpscale/packages/vdc_dashboard/bottle/deployments.py

```diff
@@ -183,7 +185,7 @@ def get_zstor_config():
         "parity_shards": 1,
         "redundant_groups": 0,
         "redundant_nodes": 0,
-        "encryption": {"algorithm": "AES", "key": "",},
+        "encryption": {"algorithm": "AES", "key": secrets.token_hex(32),},
```

Comment:
this will generate a new key every time the config is downloaded which will result in corrupt config
you can use zdb password and zfill it to 32